temporalcli/commandsmd/commands.md

```diff
@@ -1375,6 +1515,26 @@ temporal task-queue describe \
     --task-queue-type "activity"`
 ```
 
+This command provides the following task queue statistics:
+- `ApproximateBacklogCount`: The approximate number of tasks backlogged in this
+  task queue.  May count expired tasks but eventually converges to the right
+  value.
+- `ApproximateBacklogAge`: How far "behind" the task queue is running. This is
+  the difference in age between the oldest and newest tasks in the backlog,
```

Comment:
This is not accurate. The backlog age is the age of the oldest task in the backlog, period. What is written here suggest that backlog age of a backlog with a single task will be zero, as an example.